Makefile

```diff
@@ -131,7 +131,7 @@ binary: cmd/skopeo
 # Build w/o using containers
 .PHONY: bin/skopeo
 bin/skopeo:
-	$(GO) build ${GO_DYN_FLAGS} ${SKOPEO_LDFLAGS} -gcflags "$(GOGCFLAGS)" -tags "$(BUILDTAGS)" -o $@ ./cmd/skopeo
+	$(GO) build -buildvcs=false ${GO_DYN_FLAGS} ${SKOPEO_LDFLAGS} -gcflags "$(GOGCFLAGS)" -tags "$(BUILDTAGS)" -o $@ ./cmd/skopeo
```

Comment:
?

(In https://github.com/mtrmac/skopeo/tree/go1.24 I’m suggesting to rely on this data in order to simplify the `Makefile`. That might well be a bad idea for RPM packaging, I don’t know. It’s just that the change here looks so unexpected.)